VehicleManagement/Vehicle.java

```diff
@@ -0,0 +1,34 @@
+package VehicleManagement;
+
+public abstract class Vehicle {
+	private String color;
+	private String owner;
+	private double price;
+
+	public Vehicle(String color, String owner, double price) {
+		this.color = color;
+		this.owner = owner;
+		this.price = price;
+	}
+
+	public String getColor() {
+		return color;
+	}
+
+	public String getOwner() {
+		return owner;
+	}
+
+	public double getPrice() {
+		return price;
+	}
+
+	public void displayInfo() {
+		
+	}
```

Comment:
```suggestion
public abstract void displayInfo();
```
nếu method này ko có phần thân thì em mạnh dạn khai báo nó là abstract method nhé

tương tự cho method **tax**